README.md

```diff
@@ -1,7 +1,7 @@
 tessdata
 ========
 
-These language data files only work with Tesseract 4.0.0.
+These language data files only work with Tesseract 4.0.0 or 5.0.0-alpha.
```

Comment:
```suggestion
These language data files only work with Tesseract 4.0.0 an newer versions.
```

Reply:
They also work with Tesseract 4.1.0, so "newer versions" is better than adding a lengthy list of versions here.

Reply:
Sure, if that is accurate...? I only have two data points. (this is why I shouldn't submit docs updates, too - only one point of view! :-) )